packages/aws-cdk-lib/aws-ses/lib/dedicated-ip-pool.ts

```diff
@@ -24,6 +39,12 @@ export interface DedicatedIpPoolProps {
    * @default - a CloudFormation generated name
    */
   readonly dedicatedIpPoolName?: string;
+
+  /**
+   * The type of scailing mode to use for this IP pool
+   * @default ScalingMode.STANDARD
```

Comment:
```suggestion
   * The type of scaling mode to use for this IP pool.
   *
   * Updating ScalingMode doesn't require a replacement if you're updating its value from `STANDARD` to `MANAGED`.
   * However, updating ScalingMode from `MANAGED` to `STANDARD` is not supported.
   *
   * @default - `ScalingMode.STANDARD`
```
Let's provide more info.